src/main/kotlin/ResponseBuilder.kt

```diff
@@ -0,0 +1,30 @@
+package response
+
+class ResponseBuilder(private val statusCode: String, private val body: String = "") {
+    private val protocol = "HTTP/1.1"
+    private val clrf = "\r\n"
```

Comment:
`clrf` should be `crlf`. Do you know why?

Reply:
Yes, that was a typo. Per the HTTP protocol, CRLF is used to mark a line break for all protocol elements except the body.